cloud-azure/src/main/resources/definitions/azure-role-def-single-rg.json

```diff
@@ -0,0 +1,84 @@
+    "Microsoft.Network/networkInterfaces/read",
+    "Microsoft.Network/networkInterfaces/write",
+    "Microsoft.Network/networkInterfaces/delete",
+    "Microsoft.Network/networkInterfaces/join/action",
+    "Microsoft.Network/networkInterfaces/ipconfigurations/read",
+    "Microsoft.Network/networkSecurityGroups/read",
+    "Microsoft.Network/networkSecurityGroups/write",
+    "Microsoft.Network/networkSecurityGroups/delete",
+    "Microsoft.Network/networkSecurityGroups/join/action",
+    "Microsoft.Network/virtualNetworks/subnets/joinViaServiceEndpoint/action",
+    "Microsoft.Compute/availabilitySets/read",
+    "Microsoft.Compute/availabilitySets/write",
+    "Microsoft.Compute/availabilitySets/delete",
+    "Microsoft.Compute/disks/read",
+    "Microsoft.Compute/disks/write",
+    "Microsoft.Compute/disks/delete",
+    "Microsoft.Compute/images/read",
+    "Microsoft.Compute/images/write",
+    "Microsoft.Compute/images/delete",
+    "Microsoft.Compute/virtualMachines/read",
+    "Microsoft.Compute/virtualMachines/write",
+    "Microsoft.Compute/virtualMachines/delete",
+    "Microsoft.Compute/virtualMachines/start/action",
+    "Microsoft.Compute/virtualMachines/restart/action",
+    "Microsoft.Compute/virtualMachines/deallocate/action",
+    "Microsoft.Compute/virtualMachines/vmSizes/read",
+    "Microsoft.Authorization/roleAssignments/read",
+    "Microsoft.Resources/subscriptions/resourceGroups/read",
+    "Microsoft.Resources/subscriptions/resourceGroups/deployments/read",
+    "Microsoft.Resources/subscriptions/resourceGroups/deployments/write",
+    "Microsoft.Resources/subscriptions/resourcegroups/deployments/operations/read",
+    "Microsoft.Resources/subscriptions/resourcegroups/deployments/operationstatuses/read",
+    "Microsoft.Resources/deployments/read",
+    "Microsoft.Resources/deployments/write",
+    "Microsoft.Resources/deployments/delete",
+    "Microsoft.Resources/deployments/operations/read",
+    "Microsoft.Resources/deployments/operationstatuses/read",
+    "Microsoft.Resources/deployments/exportTemplate/action",
+    "Microsoft.Resources/subscriptions/read",
+    "Microsoft.ManagedIdentity/userAssignedIdentities/write",
+    "Microsoft.ManagedIdentity/userAssignedIdentities/read",
+    "Microsoft.ManagedIdentity/userAssignedIdentities/assign/action",
+    "Microsoft.DBforPostgreSQL/servers/write",
+    "Microsoft.DBforPostgreSQL/servers/delete",
+    "Microsoft.DBforPostgreSQL/servers/virtualNetworkRules/write"
+  ],
+  "NotActions": [],
+  "DataActions": [
+    "Microsoft.Storage/storageAccounts/blobServices/containers/blobs/read",
+    "Microsoft.Storage/storageAccounts/blobServices/containers/blobs/write",
+    "Microsoft.Storage/storageAccounts/blobServices/containers/blobs/delete",
+    "Microsoft.Storage/storageAccounts/blobServices/containers/blobs/add/action"
+  ],
+  "NotDataActions": [],
+  "AssignableScopes": [
+    "/subscriptions/{subscriptionId}"
```

Comment:
We should scope this on RG level, shouldn't we?

Reply:
Yes, definitely, thank you!

Reply:
Hm, but actually I am not sure if we should decrease assignableScope. What if the customer wants to use the same role definition for multiple resource groups? I think the assignableScope can be as wide as possible, whereas the real assignment should be as tight as possible.

Reply:
The customers who are asking for single RG feature will most probably scope their credential to have access on RG level, and the AssignableScopes can contain comma-separated list of RG-s